Shared/Samples/Set basemap/SetBasemapView.swift

```diff
@@ -32,19 +32,24 @@ struct SetBasemapView: View {
     }()
     
     var body: some View {
-        MapView(map: map)
-            .overlay(alignment: .topTrailing) {
-                if isShowingBasemapGallery {
-                    BasemapGallery(geoModel: map)
-                        .style(.automatic())
+        GeometryReader { geometryProxy in
+            MapView(map: map)
+                .overlay(alignment: .topTrailing) {
+                    if isShowingBasemapGallery {
+                        BasemapGallery(geoModel: map)
+                            .style(.automatic())
+                            .frame(maxWidth: geometryProxy.size.width / 3)
+                            .esriBorder()
+                            .padding()
```

Comment:
Is this padding for setting the gallery view away from the screen edge?

Reply:
Yes